aws/cloudformation/ecs/posthog.yaml

```diff
@@ -103,48 +118,123 @@ Parameters:
                  access other AWS resources like S3 buckets, DynamoDB tables, etc
   RDSNodeType:
     Description: RDS node instance class, e.g. t3.micro(free tier). See https://aws.amazon.com/rds/instance-types/
+                 If you're just testing PostHog out, the T3 is fine. If you want to run PostHog in production,
+                 you should at least run an M5. The new Graviton2 instances (M6g and R6g) are great in that they
+                 balance reduced cost for the processor with great performance.
+                 See https://aws.amazon.com/rds/postgresql/pricing/ for price information.
+                 Options are sorted by recommended type.
     Type: String
-    Default: db.t3.micro
+    Default: db.m6g.large
     ConstraintDescription: Node instance class not supported
     AllowedValues:
-      - db.t3.micro
-      - db.t3.small
-      - db.t3.medium
+      - db.m6g.large
+      - db.m6g.xlarge
+      - db.m6g.2xlarge
+      - db.m6g.4xlarge
+      - db.m6g.8xlarge
+      - db.m6g.12xlarge
+      - db.m6g.16xlarge
+      - db.r6g.large
+      - db.r6g.xlarge
+      - db.r6g.2xlarge
+      - db.r6g.4xlarge
+      - db.r6g.8xlarge
+      - db.r6g.12xlarge
+      - db.r6g.16xlarge
       - db.m5.large
       - db.m5.xlarge
       - db.m5.2xlarge
       - db.m5.4xlarge
       - db.m5.12xlarge
       - db.m5.24xlarge
+      - db.r5.large
+      - db.r5.xlarge
+      - db.r5.2xlarge
+      - db.r5.4xlarge
+      - db.r5.8xlarge
+      - db.r5.12xlarge
+      - db.r5.16xlarge
+      - db.r5.24xlarge
+      - db.t3.micro
+      - db.t3.small
+      - db.t3.medium
+      - db.t3.large
+      - db.t3.xlarge
+      - db.t3.2xlarge
+
+  RDSAllocatedStorageType:
+    Description: Type of storage engine
+    Type: String
+    Default: gp2
```

Comment:
Note (to self): this is the better default, but we need to be careful with this since changing the default here will recreate the db if anyone ever updates the CF template.

Reply:
Yeah, I had to take a snapshot and prove the `arn` to `RDSRestoreDBSnapshotId`. It worked great but it took a while.

Reply:
I added a note in the description block. I wish we could make it more prominent, but hopefully it'll draw enough attention.

Reply:
Anything we can do to make this transition smooth for other users?